src/bootstrap-decorator.js

```diff
@@ -52,4 +52,12 @@ function(decoratorsProvider, sfBuilderProvider, sfPathProvider) {
     'default': {template: base + 'default.html', builder: defaults}
   }, []);
 
-}]);
+}])
+    .controller('TextAreaController', ['$scope', 'marked', function ($scope, marked) {
+      $scope.render = function (text) {
+        if (text) {
+          return marked(text);
```

Comment:
Do you mean that you can only use this sf extension with ng-obiba, although there is no notice about that?